BackofficeBundle/Manager/NodeManager.php

```diff
@@ -92,6 +97,20 @@ public function createNewLanguageNode(NodeInterface $node, $language)
     }
 
     /**
+     * @param NodeInterface $node
+     *
+     * @return StatusInterface
+     */
+    protected function getInitialStatus(NodeInterface $node)
+    {
+        if ($node->getNodeType() == NodeInterface::TYPE_GENERAL) {
```

Comment:
Why do you make a different test here and in the transformer : 
  - One for only node transverse
  - One for all general node

What about the other nodes?